site/content/3.13/data-science/arangographml/ui.md

```diff
@@ -0,0 +1,268 @@
+    *   By aggregating feature information from this sampled neighborhood, the model creates a rich "embedding" for each node that captures both its own features and its role in the graph.
+
+## Limitations
+
+*   **Edge Attributes**: The current version of ArangoGraphML does not support the use of edge attributes as features.
+*   **Dangling Edges**: Edges that point to non-existent vertices ("dangling edges") are not caught during the featurization analysis. They may cause errors later, during the Training phase.
+*   **Prediction Scope**: Predictions can only be run in batches on the graph. It is not possible to request a prediction for a single document on-the-fly (Example, via an AQL query).
+*   **Memory Usage**: Both featurization and training can be memory-intensive. Out-of-memory errors can occur on large graphs with insufficient system resources.
+
+## The GraphML Workflow
+
+The entire process is organized into sequential steps within a **Project**, giving you a clear path from data to prediction:
+
+1.  **Featurization:** Select your data and convert it into numerical features.
+2.  **Training:** Train a GraphSAGE model on the features and graph structure.
+3.  **Model Selection:** Evaluate the trained models and choose the best one.
+4.  **Prediction:** Use the selected model to generate predictions on your data.
+5.  **Scheduling (Optional):** Automate the prediction process to run at regular intervals.
+
+## Creating a GraphML Project
+
+To create a new GraphML project using the ArangoDB Web Interface, follow these steps:
+
+1. From the left-hand sidebar, select the database where you want to create the project.
+2. In the left-hand navigation menu, click **Data Science Suite** to open the GraphML project management interface, then click Run GraphML.
+   ![Navigate to Data Science](../../../images/datascience-intro.jpg)  
+3. In the **GraphML projects** view, click  **Add new project**.
+4. The **Create ML project** modal opens. Enter a **Name** for your machine learning project.
+5. Click the **Create project** button to finalize the creation.
+6. After creation, the new project appears in the list under GraphML projects. Click the project Name to begin with a Featurization job.
+
+## Featurization Phase
+
+After clicking on a project name, you are taken to a screen where you can configure and start a new Featurization job. Follow these steps:
+- **Select a Graph** – In the **Features** section, choose your target graph from the **Select a graph** dropdown.
+- **Choose Vertex Collections** – Pick the vertex collections that you want to include for feature extraction.
+- **Select Attributes** – From the dropdown, choose the attributes from your vertex collection to convert into machine-understandable features. 
+
+{{< info >}}
+Attributes cannot be used if their values are lists or arrays.
```

Comment:
They are the same in AQL/ArangoDB Core, I'm not sure if this refers to Python or some other environment where there is a difference? One for mixed types and another where each element is of the same type??

As there is another admonition following right after, maybe put this as a regular sentence at the end of the previous bullet point?